### PR TITLE
H-624: Add GitHub issue templates

### PR DESCRIPTION
**Pull request by nonparibus**
See internal context in H-624.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Float your very best hopes and ideas
+title: ''
+labels: 'category: feature'
+assignees: ''
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/feature_request.md:6]:
```suggestion
assignees: nonparibus
```

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: 'category: bug'
+assignees: ''
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:6]:
```suggestion
assignees: CiaranMn
```

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:18]:
```suggestion
1. Click on '....'
1. Scroll down to '....'
1. See error
```

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:31]:
```suggestion
**Smartphone (please complete the following information):**

```

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:26]:
```suggestion
**Desktop (please complete the following information):**

```

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+labels: 'category: bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:14]:
```suggestion
Steps to reproduce the behavior:

```

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:35]:
```suggestion
- Device: [e.g. iPhone6]
- OS: [e.g. iOS8.1]
- Browser [e.g. stock browser, safari]
- Version [e.g. 22]
```

**Review by nonparibus — COMMENTED** [review]
Lint fixes

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:19]:
```suggestion
1.  Go to '...'
1.  Click on '....'
1.  Scroll down to '....'
1.  See error
```

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/issue-report.md:31]:
```suggestion
- OS: [e.g. iOS]
- Browser [e.g. chrome, safari]
- Version [e.g. 22]
```

**Review by TimDiekmann — APPROVED** [review]

**Pull request merged**